hnn_core/cell.py

```diff
@@ -269,13 +269,15 @@ class Cell:
         The synapses that the cell can use for connections.
     dipole_pp : list of h.Dipole()
         The Dipole objects (see dipole.mod).
-    rec_v : h.Vector()
-        Recording of somatic voltage. Must be enabled
-        by running simulate_dipole(net, record_vsoma=True)
-    rec_i : dict
-        Contains recording of somatic currents indexed
-        by synapse type. (keys are soma_gabaa, soma_gabab etc.)
-        Must be enabled by running simulate_dipole(net, record_isoma=True)
+    rec_vsec : dict
```

Comment:
should we just call it `vsec` to avoid confusion with `record_vsec` which is the boolean that decides whether to record `vsec` or not?

Reply:
Sounds good! Will be ready to merge once I push this change